docs/codeql/ql-language-reference/modules.rst

```diff
@@ -424,3 +424,30 @@ The above query therefore evalutes to:
 +----+----+
 | 4  | 4  |
 +----+----+
+
+.. index:: BigInt
+.. _bigint:
+
+BigInt
+======
+
+The built-in ``QlBuiltins`` module provides an **experimental** type ``BigInt`` of arbitrary-precision integers.
+
+This type is not available in the CodeQL CLI by default, but you can enable it by passing the ``--allow-experimental=bigint``
+option to the CodeQL CLI. Consequently, queries are currently restricted from returning BigInts as results.
+
+Unlike :code:`int` and :code:`float`, there is no automatic conversion between :code:`BigInt` and other numeric types.
+Instead, big integers can be constructed using the :code:`.toBigInt()` methods of :code:`int` and :code:`string`.
+
+The other built-in operations are:
+
+*   comparisons between :code:`BigInt`s: :code:`=`, :code:`!=`, :code:`<`, :code:`<=`, :code:`>`, :code:`>=`,
```

Comment:
GitHub's built-in RST renderer gets the quoting wrong in ``:code:`BigInt`s``. I don't know if you'll see the same in our rendered documentation. I'm sure there's a way to preview it, but I don't know how.